modules/concepts/hooks/list-of-hooks/displayProductListFunctionalButtons.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: displayProductListFunctionalButtons
```

Comment:
I checked and it seems there is a hook registered but there is no use of it anywhere in the code 🤔 
Not sure what to do about it. Maybe of it is a display hook and there is no use, we could indicate it somehow 🤔  or this could help us clean the hooks in the db

cc @jolelievre

Reply:
Same only found in some XMLs and in example module's code I wonder if we should remove from the hook.xml all the hooks that are no longer found in the code base (except for dynamic ones)

Reply:
@kpodemski Isn't this the hook to show buttons on product list hover that was removed “for performance reasons”?